mongo/instance_store.go

```diff
@@ -119,7 +119,7 @@ func (m *Mongo) UpdateDimensionNodeID(dimension *models.DimensionOption) error {
 	err := s.DB(m.Database).C(dimensionOptions).Update(bson.M{"instance_id": dimension.InstanceID, "name": dimension.Name,
 		"option": dimension.Option}, bson.M{"$set": bson.M{"node_id": &dimension.NodeID, "last_updated": time.Now().UTC()}})
```

Comment:
did you consider removing `"last_updated": time.Now().UTC()` and wrapping with `mongo.WithLastUpdatedUpdate(...)` instead?

Reply:
No, it wasn't really in scope for what I was updating (just the returned error)- handling mongo timestamps are in other stories in the backlog.